### DISCUSSION TITLE
Connecting jira url with proxy?

### DISCUSSION THREAD

**Discussion by Soinisi** [ankitpokhrel/jira-cli#303] [Q&A]
Hi, is there a setting to set proxy connection with url in the jira CLI? Many larger companies have internal proxys so if this is not possible it would be a nice feature. Thanks.

**Comment by jvtrigueros** [comment]
I believe that's already possible, if you define these variables:
```
export HTTP_PROXY=http://proxy.example.com:8080
export HTTPS_PROXY=http://proxy.example.com:8080
export NO_PROXY=localhost,127.0.0.1
```
See https://github.com/ankitpokhrel/jira-cli/blob/main/pkg/jira/client.go#L136
